src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets

```diff
@@ -69,7 +67,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       it here. The IDE is hardcoded to inject 2.0 support when needed. The settings flowing through MSBuild should reflect
       the project's runtime.
     -->
-    <RazorConfiguration Include="MVC-3.0">
+    <RazorConfiguration Include="Default" />
```

Comment:
We need to verify that tooling can work with this as well. If it shows up here, VS needs to understand it.